### DISCUSSION TITLE
supply 2 APRS-Servers simultaneously example: wettersonde.net and radiosondy.info

### DISCUSSION THREAD

**Discussion by sku2atsc** [dl9rdz/rdz_ttgo_sonde#462] [TTGO firmware]
Unfortunately, you currently have to decide whether you want to supply radiosondy or wettersonde.
Is it possible to set up 2 aprs-servers so that they can be supplied at the same time?

**Comment by dl9rdz** [comment]
yes (since dev 0814)

**Reply by SaschaSp** [reply]
Hello dl9rdz,
can you provide an example how to setup?

Many thanks

**Reply by dl9rdz** [reply]
![Screenshot 2024-09-26 at 10 48 46](https://github.com/user-attachments/assets/d2a9338d-e3a2-4680-b9a3-0450fab3edb4)

**Reply by dl9rdz** [reply]
Note that you need a new version (at least dev20240814) for that, not any of the older devel or master versions...
